packages/deployment/lib/kes.js

```diff
@@ -151,14 +152,16 @@ class UpdatedKes extends Kes {
     const src = path.join(process.cwd(), kesBuildFolder, filename);
     const dest = path.join(process.cwd(), kesBuildFolder, 'adapter', unzipFolderName);
 
-    // return Promise.resolve();
+    if (!filename) return Promise.resolve();
```

Comment:
So this would not call `super` if there is no message adapter filename? That doesn't seem right to me.

Reply:
Ah, wanted to simplify the code.  Updated.